core/new-gui/src/app/workspace/types/execute-workflow.interface.ts

```diff
@@ -91,7 +91,8 @@ export enum OperatorState {
 export interface OperatorStatistics extends Readonly<{
   operatorState: OperatorState,
   aggregatedInputRowCount: number,
-  aggregatedOutputRowCount: number
+  aggregatedOutputRowCount: number,
+  aggregatedOutputResults: ResultObject | undefined | null

```

Comment:
will this ever be undefined?

or 

will this ever be null?

Reply:
yes, non-sink operators will have null as value

Reply:
added a comment to explain it